bigbluebutton-html5/imports/ui/components/logout-confirmation/component.jsx

```diff
@@ -1,7 +1,12 @@
 import React, { Component } from 'react';
 import { withRouter } from 'react-router';
 import { defineMessages, injectIntl } from 'react-intl';
+import { makeCall } from '/imports/ui/services/api';
+import Button from '/imports/ui/components/button/component';
 import Modal from '/imports/ui/components/modal/fullscreen/component';
+import Auth from '/imports/ui/services/auth';
+import userListService from '../user-list/service';
```

Comment:
Not really. Components shouldn't hold this type of method/logic. You can find examples of that on the UserList or Chat, where the Container pass down handlers to the outer components.